app/src/main/kotlin/cz/covid19cz/erouska/exposurenotifications/ExposureNotificationsRepository.kt

```diff
@@ -86,6 +89,28 @@ class ExposureNotificationsRepository(
         }
     }
 
+    private fun setDiagnosisKeysMapping() {
+        if (System.currentTimeMillis() - prefs.getLastSetDiagnosisKeysDataMapping() > 7 * 24 * 60 * 60 * 1000) {
```

Comment:
Prosím všechny konstanty jako jak často tohle checkovat do RC

Reply:
7 dní je limit API pro tenhle call, imho není potřeba nastavovat remote

Reply:
Je to serverová věc i na straně Googlu, mohou ten limit změnit, zjistíme v produkci že se to má chovat jinak...je lepší důležité konstanty v RC. Dáme si pozor abychom to měli v RC správně.

Reply:
oka